platformio.ini

```diff
@@ -50,12 +57,16 @@ build_flags =
 	;-DRF_MODULE_GDO2=RADIOLIB_NC    ; CC1101 pin GDO2
 	;-DRF_MODULE_INIT_STATUS=true
 
+extra_scripts = 
+	pre:patch.py
```

Comment:
This patch had already been applied in the framework

Reply:
The less forking the better should this project get a bit larger.